app/src/features/surveys/observations/sampling-sites/SamplingSiteList.tsx

```diff
@@ -250,12 +282,43 @@ const SamplingSiteList = () => {
               ml: 1,
               mr: -1
             }}
-            aria-label="bulk delete"
+            aria-label="header-settings"
             disabled={!checkboxSelectedIds.length}
-            onClick={handlePromptConfirmBulkDelete}>
-            <Icon path={mdiTrashCanOutline} size={1} />
+            onClick={handleHeaderMenuClick}>
+            <Icon path={mdiDotsVertical} size={1} />
           </IconButton>
         </Toolbar>
+        <Divider />
+        <Box
+          display="flex"
+          overflow="hidden"
+          alignItems="center"
+          pl={3}
+          pr={2.5}
+          height={55}
+          className="sampleSiteHeader">
+          <Checkbox
+            checked={checkboxSelectedIds.length === samplingSiteCount}
```

Comment:
I think the checkbox should be indeterminate if some (but not all) of the records are selected.

```
<Checkbox
  checked={checkboxSelectedIds.length === samplingSiteCount}
  indeterminate={checkboxSelectedIds.length > 1 && checkboxSelectedIds.length < samplingSiteCoutn}
  ...
/>